dune_project.ml

```diff
@@ -1,76 +1,74 @@
 open Types
 
-type t = Dune_lang.t list
+module Deps = struct
+  include Deps
+end
```

Comment:
This is not necessary and could be removed by tweaking a bit the module definitions. Or with this, if I'm not mistaken.
```suggestion
module Deps = Deps
```